jquery.flot.touchNavigate.js

```diff
@@ -168,16 +169,21 @@
             zoomAmount = pinchDistance(e) / gestureState.prevDistance,
             dist = pinchDistance(e);
 
-        center.left = getPoint(e, 'pinch').x - offset.left;
-        center.top = getPoint(e, 'pinch').y - offset.top;
+        if (gestureState.twoTouches || Math.abs(dist - gestureState.prevDistance) > DISTANCE_MARGIN) {
+            center.left = getPoint(e, 'pinch').x - offset.left;
+            center.top = getPoint(e, 'pinch').y - offset.top;
 
-        // send the computed touched axis to the zoom function so that it only zooms on that one
-        plot.zoom({
-            center: center,
-            amount: zoomAmount,
-            axes: navigationState.touchedAxis
-        });
-        gestureState.prevDistance = dist;
+            // send the computed touched axis to the zoom function so that it only zooms on that one
+            plot.zoom({
+                center: center,
+                amount: zoomAmount,
+                axes: navigationState.touchedAxis
+            });
+            gestureState.prevDistance = dist;
+
+            //activate zomm mode
+            gestureState.twoTouches = true;
```

Comment:
This assignment should be moved to pinch.drag() as well.

The name is a bit confusing since the pinch.drag() is called when 2 touches are pressed. Maybe zoomEnabled?